server/subscriptions.go

```diff
@@ -31,13 +31,13 @@ func (p *Plugin) Subscribe(ctx context.Context, bitbucketClient *bitbucket.APICl
 	var err error
 
 	if repo == "" {
-		_, _, err = bitbucketClient.UsersApi.UserGet(ctx)
+		_, _, err = bitbucketClient.UsersApi.UserGet(ctx) //nolint:bodyclose
```

Comment:
What does this mean? What body in this context _could_ be closed that is not being closed?

Reply:
It's a false positive. `bitbucketClient.UsersApi.UserGet` returns a response body, but takes care of closing it itself.